pygments/lexers/lilypond.py

```diff
@@ -169,7 +169,9 @@ def get_tokens_unprocessed(self, text):
             (r"([^\W\d]|-)+(?=([^\W\d]|[\-.])*\s*=)", Token.Name.Lvalue),
 
             # Virtually everything can appear in markup mode, so we highlight
-            # as text.
+            # as text.  Try to get a complete word, or we might wrongly lex
+            # a suffix that happens to be a builtin as a builtin (e.g., "myStaff").
+            (r"([^\W\d]|-)+?" + NAME_END_RE, Token.Text),
```

Comment:
so that's `[a-zA-Z]` but Unicode enabled? Took me a while to get through the negatives :)

Reply:
Ha, one more reason to switch to `regex` :-)

Reply:
yes sure!

Reply:
Thanks, merging then.